File: hellocharts/src/main/java/net/raquezha/lecho/hellocharts/listener/ColumnChartOnValueSelectListener.java
package net.raquezha.lecho.hellocharts.listener;


import net.raquezha.lecho.hellocharts.model.SubcolumnValue;

public interface ColumnChartOnValueSelectListener extends OnValueDeselectListener {

    void onValueSelected(int columnIndex, int subcolumnIndex, SubcolumnValue value);

}

File: hellocharts/src/main/java/lecho/lib/hellocharts/listener/ColumnChartOnValueSelectListener.kt
package lecho.lib.hellocharts.listener

import lecho.lib.hellocharts.model.SubcolumnValue
import lecho.lib.hellocharts.model.TouchCoordinates

interface ColumnChartOnValueSelectListener : OnValueDeselectListener {

    fun onValueSelected(
        columnIndex: Int,
        subColumnIndex: Int,
        value: SubcolumnValue,
        touchCoordinates: TouchCoordinates?
    )
}